systems/riken-fugaku/system.py

```diff
@@ -5,11 +5,29 @@
 
 import pathlib
 
-from benchpark.directives import variant
+from benchpark.directives import variant, maintainers
 from benchpark.system import System
+from benchpark.paths import hardware_descriptions
 
 
 class RikenFugaku(System):
+    id_to_resources = {
+        "fugaku": {
+            "sys_cores_per_node": 48,
+            "sys_mem_per_node": 32,
+            "hardware_key": str(hardware_descriptions)
+            + "/Fujitsu-A64FX-TofuD/hardware_description.yaml",
+        },
+    }
+
+    system_site = "riken"
+
+    variant(
```

Comment:
why do we have a variant with a single possible value?

Reply:
Removed in [c8e9fcf](https://github.com/LLNL/benchpark/pull/587/commits/c8e9fcfe4a1aa7525418ba797baeed6cb4656366)